src/controllers/error.controller.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response } from "express";
+import { Templates } from "../types/template.paths";
+
+const pageNotFound = (req: Request, res: Response) => {
+  return res.status(404).render(Templates.ERROR_404);
+};
+
+export default pageNotFound;
```

Comment:
just thinking ahead to when we will need to also have an "500 Something went wrong" page.
It may be best not to have a default export on this file.

Export the pageNotFound function like so: `export const pageNotFound = ...`
And on app.ts import it like `import { pageNotFound } from "./controllers/error.controller";`
And use it like `app.use(pageNotFound);`

This way we can just add an "internalServerError" function to the error controller class without breaking anything

Reply:
I can make that change if you think it's best, i used:
https://github.com/companieshouse/strike-off-objections-web/blob/master/src/controllers/error.controller.ts
As a basis and in that file it looked like the export could be updated later on to include other errors. Unless i'm miss understanding how that works.

Reply:
Ah yes I see how that works now, in that case leave it be this will work well, i'll approve